tutorial/markdown/kotlin/ktor.md

```diff
@@ -102,195 +95,356 @@ fun createScope(bucket: Bucket, configuration: CouchbaseConfiguration): Scope {
+│   │   │   │   ├── AirportModel.kt
+│   │   │   │   └── RouteModel.kt
+│   │   │   ├── repositories
+│   │   │   │   ├── AirlineRepository
+│   │   │   │   ├── AirportRepository
+│   │   │   │   └── RouteRepository
+│   │   │   ├── routes
+│   │   │   │   ├── AirlineRoutes.kt
+│   │   │   │   ├── AirportRoutes.kt
+│   │   │   │   └── RouteRoutes.kt
+│   │   │   ├── services
+│   │   │   │   ├── AirlineService
+│   │   │   │   ├── AirportService
+│   │   │   │   └── RouteService
+│   │   │   ├── Application.kt
+│   │   │   └── CouchbaseConfiguration.kt 
+│   │   └── resources
+│   │       ├── application.conf
+│   │       └── logback.xml
+│   └── test/kotlin/com/couchbase
+│       └── kotlin/quickstart
+│           ├── AirlineTests
+│           ├── AirportTests
+│           └── RouteTests
+├── build.gradle.kts
+└── Dockerfile
 ```
-> _from [`src/main/kotlin/com/couchbase/kotlin/quickstart/ProfileRepository.kt`](https://github.com/couchbase-examples/kotlin-quickstart/blob/main/src/main/kotlin/com/couchbase/kotlin/quickstart/ProfileRepository.kt)_
 
-The repository method creates new Profile object with random UUID, populates it with received data and then uses SDK collection object to store the profile on the cluster.
+### Airport Entity
 
-Stored profile is then returned up the call stack and rendered as JSON in HTTP response body.
+For this tutorial, we will focus on the airport entity. The other entities are similar.
 
-## Working with existing Profiles
+We will be setting up a REST API to manage airport documents.
 
-### Listing profiles
+- [POST Airport](#post-airport) – Create a new airport
+- [GET Airport](#get-airport) – Read specified airport
+- [PUT Airport](#put-airport) – Update specified airport
+- [DELETE Airport](#delete-airport) – Delete airport
+- [Airport List](#list-airport) – Get all airports. Optionally filter the list by country
+- [Direct Connections](#direct-connections) - Get a list of airports directly connected to the specified airport
 
-The next handler function processes HTTP GET requests to the `/profile` API endpoint.
-Clients can use this endpoint to request all previously created profiles.
-The function accepts a `PaginationRequest` object, which represents a set of GET parameters:
-```
-data class PaginationRequest (
-    @QueryParam("Profiles to skip before the results", allowEmptyValues = true)
-    val skip: Int?,
-    @QueryParam("Number of results", allowEmptyValues = true)
-    val limit: Int?
-)
+For CRUD operations, we will use the [Key-Value operations](https://docs.couchbase.com/dotnet-sdk/current/howtos/kv-operations.html) that are built into the Couchbase SDK to create, read, update, and delete a document. Every document will need an ID (similar to a primary key in other databases) to save it to the database. This ID is passed in the URL. For other end points, we will use [SQL++](https://docs.couchbase.com/dotnet-sdk/current/howtos/n1ql-queries-with-sdk.html) to query for documents.
```

Comment:
The links are referring to dotnet in this paragraph

Reply:
corrected it.